docs/source/en/using-diffusers/loading_adapters.md

```diff
@@ -506,22 +506,11 @@ import torch
 from diffusers import StableDiffusionPipeline, DDIMScheduler
 from diffusers.utils import load_image
 
-noise_scheduler = DDIMScheduler(
-    num_train_timesteps=1000,
-    beta_start=0.00085,
-    beta_end=0.012,
-    beta_schedule="scaled_linear",
-    clip_sample=False,
-    set_alpha_to_one=False,
-    steps_offset=1
-)
-
 pipeline = StableDiffusionPipeline.from_pretrained(
     "runwayml/stable-diffusion-v1-5",
     torch_dtype=torch.float16,
-    scheduler=noise_scheduler,
 ).to("cuda")
```

Comment:
This pipeline isn't populated with `image_encoder`. Running the code snippet as is will lead to errors here no? I think we'd want these guides to be as self-contained as possible, meaning if a user takes the code snippet, it should run without errors. This is why I opened https://github.com/huggingface/diffusers/pull/6597.

Reply:
ohhhh - you don't need image_encoder in the pipeline; it loads from the ip-adapter repo instead https://huggingface.co/h94/IP-Adapter/tree/main/models/image_encoder

Reply:
Yes, I closed the PR after realizing it. Sorry.